pages/_app.tsx

```diff
@@ -67,11 +66,10 @@ export default class MyApp extends App {
         return <Unauthorized/>;
       } else {
         this.redirect("/authenticate");
-
         return <Redirect/>;
       }
     } else {
-      return <Loader/>;
+      return <Loader />;
```

Comment:
Did you run the linter? Wonder if the linter is not picking this space up.

Reply:
I didn't run yarn lint. I thought I have lint-on-save configured (both in VS Code and I also think in the next webpack config). I'm testing and will make sure to run a yarn lint before submitting. 

I'll let you know if there are any issues in the config. Right now I can't save without the space being removed and receiving a warning.